lib/apruve/resources/corporate_account.rb

```diff
@@ -3,9 +3,17 @@ class CorporateAccount < Apruve::ApruveObject
     attr_accessor :id, :merchant_uuid, :customer_uuid, :type, :created_at, :updated_at, :payment_term_strategy_name,
                   :disabled_at, :name, :creditor_term_id, :payment_method_id, :status, :trusted_merchant
 
```

Comment:
I just noticed that this class doesn't include `authorized_buyers`.  That information is needed for the Quickbooks integration.